policy-design-lab/pdl-frontend
# Update CRP, ACRP,RCPP and Crop Insurance to fit the new data structure

This PR is for #320, #322, #324, and #325. It updates the CRP, ACEP, RCPP, and Crop Insurance to the new data structure and replaces the treemap on the ACEP page with a table, as requested by the collaborator on Slack. The collaborator has verified the data on the dev site.

## How to Test
1. Navigate to https://policydesignlab-dev.ncsa.illinois.edu/.
2. Check CRP, ACEP, RCPP, and Crop Insurance to ensure they fit the data returned by the API endpoint.
3. Navigate to the ACEP page and confirm that the table has been updated to match the style of the RCPP page: <img width="1380" alt="Screenshot 2024-10-02 at 5 22 33 PM" src="https://github.com/user-attachments/assets/2f487b06-8437-4d3c-b4bf-75c0c85af65b">
